README.md

```diff
@@ -135,6 +135,35 @@ npx react-native link
 > For component detailed API and usage examples, check out the page [Using NatDS Icons](https://natds-rn.natura.design/?path=/docs/documentation-icons--page/)
 > and the [Icon component documentation](https://natds-rn.natura.design/?path=/docs/components-icon--default/).
 
+## Fonts
+
+We have the brand fonts package `@naturacosmeticos/natds-themes` as a dependency of this library, so you have the brand fonts files available in your `node_modules` folder after installation.
```

Comment:
```suggestion
The brand fonts package `@naturacosmeticos/natds-themes` is a dependency of this library, so the font files by brand are available in the `node_modules` folder after installation. To use them, import the method `copyFontsByBrand` and add their path to the `react-native.config.js` file:
```

Reply:
Fiz um merge entre as duas sugestões, oque acham?

The brand fonts package `@naturacosmeticos/natds-themes` is a dependency of this library, so the font files by brand are available in the `node_modules` folder after installation. To do this, import the method `copyFontsByBrand` to `react-native.config.js` file, passing the brand that you need and the path to your assets folder: